WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -1016,6 +1032,13 @@ extension AztecPostViewController {
             _ = navigationController?.popViewController(animated: true)
         }
     }
+
+    func shouldRemoveOnDismiss(post: AbstractPost) -> Bool {
+        let originalTitleIsEmpty = post.original?.postTitle?.isEmpty ?? true
+        let originalContentIsEmpty = post.original?.content?.isEmpty ?? true
+
+        return post.isRevision() && post.hasLocalChanges() && originalTitleIsEmpty && originalContentIsEmpty
```

Comment:
I'm not sure these conditions are correct / complete.

I've tested this PR by creating a new post, then typing "A" as the title, then discarding it.  The post was not removed as I'd expect.